src/templates/project/components/about/index.tsx

```diff
@@ -18,7 +26,13 @@ const AboutProject: React.FC<Props> = ({ volunteers, description }) => {
       <Heading2>
         {volunteers.length}&nbsp;{t('pages.project.about.volunteers.title')}
       </Heading2>
-      <Body>{t('pages.project.about.volunteers.subtitle')}</Body>
+      <Body>
+        {getBodyText({
```

Comment:
Je treba mit to jako helper function getBodyText, ktera checkuje stejne jen if condition? Nestaci nam na to ternary condition?

```
{progress >= FINISHED_PROJECT_PROGRESS ? finishedProjectSubtitle  : t('pages.project.about.volunteers.subtitle')}
```

Reply:
Dokáži si představit obojí, ale udělal jsem funkci z důvodů, že je zde několik věcí najednou: 

- Podmíněný výsledek (if true, else)
- Samotná podmínka (kdy vrátit co) 
- Překlad

Pro jednodušší věci lze určitě použít ternární operátor.